layouts/page/virtual-backgrounds.html

```diff
@@ -0,0 +1,60 @@
+{{ define "hero" }}
+    <header>
+        <div class="container mx-auto p-4 text-center flex-col m-24 mb-12">
+            <h1 class="text-5xl">{{ .Params.hero.title }}</h1>
+            <p class="mt-10 text-2xl">{{ .Params.hero.description }}</p>
+        </div>
+    </header>
+{{ end }}
+
+{{ define "main" }}
+    <section id="backgrounds" class="container mx-auto">
+        {{ range .Params.backgrounds }}
+            <div class="flex flex-col lg:flex-row my-4 lg:my-12">
+                <div class="w-full mb-12 order-last lg:order-first lg:mb-0 lg:w-5/12 p-4 px-16">
+                    <p class="text-2xl font-bold">{{ .name }}</p>
+                    <p class="mb-12">{{ .description }}</p>
+                    <a href="{{ .image }}" class="btn btn-lg" download>
+                        Download
+                    </a>
+                </div>
+                <div class="w-full order-first lg:order-last lg:w-7/12">
+                    <img src="{{ .image }}" alt="{{ .name }}" />
+                </div>
+            </div>
+        {{ end }}
+    </section>
+
+    <section id="get-started" class="bg-gray-100 py-16 px-4 lg:px-0">
+        <div class="container mx-auto max-w-5xl md:flex my-8 align-top justify-center text-center">
+            <div class="md:w3-/12 mr-4">
+                <h2>Get Started with Pulumi</h2>
+                <p class="text-lg">
+                    Use Pulumi's open source SDK to create, deploy, and manage infrastructure on any cloud.
+                </p>
+            </div>
+        </div>
+        <div class="container mx-auto max-w-5xl">
+            <div class="flex flex-row max-w-full">
```

Comment:
```suggestion
            <div class="flex max-w-full">
```

You can delete these, since `row` is the default.

Reply:
👍